README.md

```diff
@@ -32,7 +32,13 @@ The Python interpreter is not required in that case.
 - Maven 3.0.0 or newer
 - Python 3.9 or newer
 - [tox](https://tox.readthedocs.io/en/latest/) - `pip install tox`
-- Run `git submodule update --init` to retrieve [Typeshed](https://github.com/python/typeshed) as a Git submodule
+- Run `git submodule update --init` to retrieve [Typeshed](https://github.com/python/typeshed) and [SKlearn stubs](https://github.com/microsoft/python-type-stubs/tree/main/stubs/sklearn) as a Git submodules.
+- Run the following commands to extract only the needed files for the SKlearn stubs
+```
+cd python-frontend/typeshed_serializer/resources/python-type-stubs
+git sparse-checkout set --no-cone stubs/sklearn 
+git checkout
```

Comment:
It works, but it is quite convoluted. If you do the steps incorrectly, you can get stuck in a sparse checkout and it is not obvious how to fix it.

Reply:
I know it is not perfect sadly there is no easier way. I think we could maybe automate the process, in a maven task or small script. But I think this could be a separate ticket/PR